en/mkdocs.yml

```diff
@@ -544,12 +544,14 @@ nav:
                 - Handling Errors: use-cases/streaming-usecase/handling-errors.md
                 - Performing ETL Operations: use-cases/streaming-usecase/performing-etl-tasks.md
                 - Triggering Integration Flows: use-cases/streaming-usecase/triggering-integration-flows.md
+                - Exposing Stream as Managed API: use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalogue.md
```

Comment:
I moved the 'Exposing Integration Service as Managed API' doc to the top level of the tutorials.